app/assets/javascripts/app/views/publisher/uploader_view.js

```diff
@@ -76,7 +76,9 @@ app.views.PublisherUploader = Backbone.View.extend({
       .prepend(
         "<div class=\"x\"></div>" +
         "<div class=\"circle\"></div>" +
-        "<img src=\"" + image.url + "\" data-id=\"" + photoId + "\" alt=\"\" class=\"hidden\" />"
+        "<img src=\"" + image.thumb_medium.url + "\" data-id=\"" + photoId + "\"" +
+        " data-small=\"" + image.thumb_small.url + "\" data-scaled=\"" + image.scaled_full.url + "\"" +
+        " class=\"hidden\" />"
```

Comment:
This... hurts my eyes. Can we replace this with something like...

```js
    var imgElement = document.createElement("img");
    imgElement.src = image.thumb_medium.url;
    imgElement.classList.add("hidden");
    imgElement.dataset.id = photoId;
    imgElement.dataset.small = image.thumb_small.url;
    imgElement.dataset.scaled = image.scaled_full.url;

    placeholder
      .prepend(
        '<div class="x"></div>' +
        '<div class="circle"></div>' +
        imgElement.outerHTML
      ).removeClass("loading");
```

It is more verbose, but also a lot more readable imho.

Reply:
Fixed :+1: